src/Tizen.Multimedia.Remoting/MediaController/MediaControlCommand.cs

```diff
@@ -59,10 +60,22 @@ internal void SetRequestInformation(string receiverId)
         /// <param name="requestId">The request Id for each command.</param>
         internal void SetResponseInformation(string receiverId, string requestId)
         {
-            ReceiverId = receiverId ?? throw new ArgumentNullException(nameof(receiverId)); ;
-            _requestId = requestId ?? throw new ArgumentNullException(nameof(requestId)); ;
+            ReceiverId = receiverId ?? throw new ArgumentNullException(nameof(receiverId));
+            _requestId = requestId;
+
+            if (_requestId == null)
+            {
+                Log.Info(GetType().FullName, "request_id is null. Response() should not be called.");
+            }
         }
 
+        /// <summary>
+        /// Gets the status of response.
+        /// </summary>
+        /// <remarks>If false, the receiver should not response for the received command.</remarks>
+        /// <since_tizen> 12 </since_tizen>
+        public bool NeedToResponse => _requestId != null;
```

Comment:
please change the name of this property, like "IsResponseNeeded", "ResponseNeeded", "NeedToRespond", it cannot be [to + noun]

Reply:
I'll change it to 'IsResponseNeeded'. But 'NeedToResponse' is already used in VD, so I'll keep it internal api only.